server-master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -193,7 +193,13 @@ private[celeborn] class Master(
       executeWithLeaderChecker(null, timeoutDeadWorkers())
     case CheckForApplicationTimeOut =>
       executeWithLeaderChecker(null, timeoutDeadApplications())
-    case WorkerLost(host, rpcPort, pushPort, fetchPort, replicatePort, requestId) =>
+    case pb: PbWorkerLost =>
+      val host = pb.getRequestId
```

Comment:
pb.getHost

Reply:
fixed, but I can't believe UT passed w/ such error... we really should improve UT

Reply:
+1